### DISCUSSION TITLE
OKEX模块的使用

### DISCUSSION THREAD

**Discussion by zweix123** [veighna-global/vnpy_evo#24] [Q&A]
请问怎么使用OKEN模块进行模拟交易？
我在OkexGateway类中填写了default_setting字段（没有填写代理地址和代理端口），但是其他trader进行连接时会显示私有频道、公有频道触发异常。

**Comment by zweix123** [comment]
范围缩小了，使用公司网没有问题，但是使用代理就不行，是全局代理，配置文件是我个人写的；但是即使用别人提供的配置文件也不行。报错“私有频道触发异常，公有频道触发异常”

**Comment by zweix123** [comment]
通过使用海外服务器解决。

**Comment by zweix123** [comment]
请问怎么使用vnpy的OKX交易呢？这方面并没有相关手册。

**Reply by menglingpu** [reply]
你好，我也遇到了这个问题，请问您解决了吗

**Reply by zweix123** [reply]
> 你好，我也遇到了这个问题，请问您解决了吗

抱歉，我后面不再使用vnpy。假如您的目的是OKX交易，而不是将vnpy上的策略应用在OKX上，可以使用其他OKX的客户端（当然我这里也没有推荐）

**Reply by menglingpu** [reply]
就是想将vnpy上的策略应用在OKX上
